categories.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
why is this file needed? does it have to be in the project root?
I'm starting to think we might need a separate project for this, or use a different approach, e.g. fork their repo

Reply:
done